src/renderer/component/cardVerify/view.jsx

```diff
@@ -7,6 +7,20 @@ let scriptLoading = false;
 let scriptLoaded = false;
 let scriptDidError = false;
 
+declare class CardVerify {
+  static stripeHandler: {
+    open: Function,
+    close: Function,
+  };
+}
+
+declare class StripeCheckout {
+  static configure({}): {
+    open: Function,
+    close: Function,
+  };
+}
+
```

Comment:
We don't need these. They aren't doing anything as is. Also I belive the `declare class` syntax is wrong.

Reply:
Thank you for your feedback.

The classes are types for the global `CardVerify` object that you are accessing after adding the script. Not declaring classes for them results in Flow errors

`Cannot resolve name CardVerify.`
`Cannot resolve name StripeCheckout.`

As this is a global variable, it would make sense in my opinion to also declare a global class like suggested here: https://flow.org/en/docs/libdefs/creation/#toc-declaring-a-global-class

The goal of my changes was to eliminate the lint & flow errors and therefore I added class definitions for the accessed global objects.

Reply:
Ah you are right! I missed the part where we access `CardVerify` when I first looked this over. Thanks for that link too 🙂 I'll definitely read over that.